README.md

```diff
@@ -45,6 +45,18 @@ To bootstrap and compile it at the same time,
 $ python3 apertium-init.py foo --analyser=hfst && ( cd apertium-foo/ && ./autogen.sh && make )
 ```
 
+To include a twoc file for handling prefixes,
```

Comment:
Didn't we say it'd be `.spellrelax` ?

Reply:
I'm not sure what you're referring to. This example is for the `--with-twoc` option.

Reply:
Ah, sorry, I was getting the two twoc files confused.